gitlab.go

```diff
@@ -52,6 +52,22 @@ const (
 	OwnerPermission      AccessLevel = 50
 )
 
+// NotificationLevel represents a notification level within Gitlab.
+//
+// GitLab API docs: http://doc.gitlab.com/ce/...?
+type NotificationLevel int
+
+// List of available notification levels
+//
+// GitLab API docs: http://doc.gitlab.com/ce/...?
+const (
+	DisabledNotifications NotificationLevel = 0
+	ParticipatingNotifications NotificationLevel = 1
+	WatchNotifications NotificationLevel = 2
+	GlobalNotifications NotificationLevel = 3
+	MentionNotifications NotificationLevel = 4
+)
```

Comment:
I found it only in the code:
https://github.com/gitlabhq/gitlabhq/blob/master/app/models/notification.rb#L3